src/System.Private.CoreLib/shared/Interop/Windows/Kernel32/Interop.HandleTypes.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+        internal partial class HandleTypes
+        {
+            internal const int STD_INPUT_HANDLE = -10;
```

Comment:
Why have these constants in both `Interop.Kernel32` (Interop.GetStdHandle.cs) and `Interop.Kernel32.HandleTypes` (Interop.HandleTypes.cs)?

Reply:
This file wasn't meant to be included; note that it's not included in any project.  Will delete it.

Reply:
Hmm. Looking at the project, I can't see this file included anywhere. Maybe it's dead code?